### PR TITLE
DTGB-406: Changes for the footer.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Review by delrueba — DISMISSED** [review]
You're going for a darker background,
did you recheck contrast?

**Review by Jeroen005 — APPROVED** [review]

**Comment by meirege** [comment]
@delrueba You're right it's not wcag compliant :(

**Review by Jeroen005 — DISMISSED** [review]
😭

File: starterkit/starterkit.info.yml [hunk 1]
```diff
@@ -19,5 +19,4 @@ regions:
   footer_2: Footer middle left
   footer_3: Footer middle right
   footer_4: Footer right
-  footer_5: Footer bottom left
-  footer_6: Footer bottom right
+  footer_5: Footer bottom right
```
Comment (Jeroen005) [inline starterkit/starterkit.info.yml]: starterkit en gent_base lijken niet gelijk te lopen, correct?

File: styleguide/components/11-base/collections/_collections.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.collection {
```
Comment (Jeroen005) [inline styleguide/components/11-base/collections/_collections.scss:1]: What is a collection?

File: starterkit/starterkit.info.yml (hunk 1 above)
Comment (meirege) [reply starterkit/starterkit.info.yml]: Apparently, I updated starterkit.

File: styleguide/components/11-base/collections/_collections.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/11-base/collections/_collections.scss:1]: It's a class name to use for collections of data, like views in Drupal.

**Comment by meirege** [comment]
@delrueba Link color has been changed to a darker tint to be wcag compliant.

File: styleguide/components/11-base/collections/_collections.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/11-base/collections/_collections.scss:1]: @Jeroen005 Placeholder was added to provide collection styling to other elements should it be necessary in the future.

**Review by delrueba — APPROVED** [review]

**Pull request merged**